docs/Users-Guide.md

```diff
@@ -235,84 +324,151 @@ Parameters:
 
 The sketch id is inferred from the filename if it starts with a number. The timeline name can also be generated from the filename if not specified.
 
-
 ### similarity_score
 
 Command:
-```
+
+```shell
 tsctl similarity_score
 ```
 
-## Concepts
+## Searching
 
-Timesketch is built on multiple sketches, where one sketch is usually one case.
-Every sketch can consist of multiple timelines with multiple views.
+There is a dedicated document called [SearchQueryGuide](/docs/SearchQueryGuide.md) to help you create custom searches.
 
-### Sketches
+All data within Timesketch is stored in elasticsearch. So the search works similar to ES.
 
-There is a dedicated document to walk you through [Sketches](/docs/SketchOverview.md)
+Using the advances search, a JSON can be passed to Timesketch
 
-### Adding Timelines
+```json
+{
+ "query": {
+   "bool": {
+     "must": [
+       {
+         "query_string": {
+           "query": "*"
+         }
+       }
+     ]
+   }
+ },
+ "sort": {
+   "datetime": "asc"
+ }
+}
+```
 
-* [Create timeline from JSON/JSONL/CSV file](/docs/CreateTimelineFromJSONorCSV.md)
-* [Create timeline from Plaso file](/docs/CreateTimelineFromPlaso.md)
-* [Enable Plaso upload via HTTP](/docs/EnablePlasoUpload.md)
+## Analyzers
 
-### Adding event
+With Analyzers you can enrich your data in timelines. The analysers are written in Python.
 
-*This feature is currently not implemented in the Web UI*
+The system consist of a set of background workers that can execute Python code on a stream of events. It provides an easy to use API to programmatically do all the actions available in the UI, e.g. tagging events, star and create saved views etc. The idea is to automatically enrich data with encoded analysis knowledge.
 
-~~To manually adding an event, visit the sketch view. Within that screen, there is the possibility to star an event, hide an event as well as add a manual event (marked with a little +).
-This event will have the previously selected time pre-filled but can be changed.~~
+The code for Analyzers is located at
 
-### Views
+```shell
+/timesketch/lib/analyzers
+```
 
-#### Hiding events from a view
+### Develop Analyzers
 
-All about reducing noise in the result views.
-Hit the little eye to hide events from the list making it possible to
-curate views to emphasize the important things.
-The events are still there and can be easily shown for those who want to see them.
-Hit the big red button to show/hide the events.
+There is a dedicated document to walk you through the process of developing [Analyzers](/docs/WriteAnalyzers.md)
 
-### Heatmap
+### Analyzer description
 
-*This feature is currently not implemented in the Web UI*
+*Note:* Not all analyzers are explained in this documentation. If you have questions to a particular analyzer, please have a look at the code or file an Issue on Github.
```

Comment:
can you make the "Issue on Github" a link to file a new issue?

Reply:
easy